<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView"/>
      <widget class="QTextEdit" name="textEdit"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddSibling"/>
    <addaction name="actionAddChild"/>
    <addaction name="actionRemoveRequirement"/>
    <addaction name="actionProjectSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionAddSibling"/>
   <addaction name="actionAddChild"/>
   <addaction name="actionRemoveRequirement"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionBulletList"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/document-open-3.ico</normaloff>:/icons/Resources/document-open-3.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Requirements File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/document-save-as-5.ico</normaloff>:/icons/Resources/document-save-as-5.ico</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save Requirements as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/document-save-5.ico</normaloff>:/icons/Resources/document-save-5.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Requirements</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/format-text-italic-2.ico</normaloff>:/icons/Resources/format-text-italic-2.ico</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="toolTip">
    <string>Italic Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/format-text-bold-2.ico</normaloff>:/icons/Resources/format-text-bold-2.ico</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Bold Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionBulletList">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/format-list-unordered.ico</normaloff>:/icons/Resources/format-list-unordered.ico</iconset>
   </property>
   <property name="text">
    <string>Bullet List</string>
   </property>
   <property name="toolTip">
    <string>Format as bullet list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAddSibling">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/list-add.ico</normaloff>:/icons/Resources/list-add.ico</iconset>
   </property>
   <property name="text">
    <string>Add Requirement</string>
   </property>
   <property name="toolTip">
    <string>Append Requirement</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAddChild">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/list-add-3.ico</normaloff>:/icons/Resources/list-add-3.ico</iconset>
   </property>
   <property name="text">
    <string>Add Subrequirement</string>
   </property>
   <property name="toolTip">
    <string>Add Subrequirement</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionProjectSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/system-settings.ico</normaloff>:/icons/Resources/system-settings.ico</iconset>
   </property>
   <property name="text">
    <string>Project settings...</string>
   </property>
  </action>
  <action name="actionRemoveRequirement">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Resources/list-remove-5.ico</normaloff>:/icons/Resources/list-remove-5.ico</iconset>
   </property>
   <property name="text">
    <string>Remove Requirement</string>
   </property>
   <property name="toolTip">
    <string>Remove Requirement</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
